src/openapi.json

```diff
@@ -38,6 +38,16 @@
         "name": "_content",
         "required": false,
         "description": "A web-search formatted string to select specific entities returned."
+      },
+      "opportunityIdParam": {
+        "schema": {
+          "type": "integer",
+          "default": 0
```

Comment:
Having a default of `0` feels very strange to me. I recognize that this is extending the pattern for the other query parameters on this endpoint, but it feels different.

One way I look at default values is by rewriting the call with all the values specified. That'd transform `/proposals` to `/proposals?_page=1&_count=100&_content=&opportunityId=0`.

The page and count parameters must have some kind of default, as they are required for processing a request, and the alternative is to make them non-optional.

The search parameter is optional, and its (non-empty) presence significantly affects the results; making it default to an empty string is a little bit weird, but it makes some sense: `?_content=` is clearly missing a value.

Perhaps the reason setting the default to `0` here is strange is because `0` is conceivably in the domain of opportunity IDs. Opportunity IDs are integers, 0 is an integer, therefore 0 could be an opportunity ID. This is incorrect, of course, but I feel like setting this default value - and communicating it in the documentation - encourages this confusion.

As an alternative, can we not set a default value, properly detect the parameter's presence or absence, and handle it appropriately?

Reply:
Yes, it is strange. I followed the established pattern but agree it feels different for an integer. The root of the default I think comes from wanting to have a single database query for `GET /proposals` with potentially many parameters. But as you mentioned, perhaps we can default somewhere between the client API call and the database code.

Reply:
Could we just specify that the default is null?

Reply:
Having a default specified for search parameters also feels weird to me, yet [that is what happens](https://github.com/PhilanthropyDataCommons/service/blame/59e39aaae30d1dfd93261dfa11795ea155cc7d58/src/queryParameters/extractSearchParameters.ts#L10) and was approved in #320. Is that worth revisiting too?

Reply:
I'd be amenable to revisiting `_content`, but as I said it feels different. Are you offering to tackle that as part of this PR, @bickelj?

Reply:
Probably not but maybe it will be simple enough to include. I will revisit when refactoring the one under discussion here.